src/main/resources/snippets/jython/dispatch.py

```diff
@@ -0,0 +1,5 @@
+def dispatch(data):
+    result = ""
+    for item in data:
+        result = result + item.__str__()
```

Comment:
Hum the goal is to measure the cost of method dispatch, hence `str(...)` is probably not the best fit. Or is it delegating to `__str__()` underneath? (in which case it would still be an extra call)